homework/index.js

```diff
@@ -36,7 +36,44 @@
       if (err) {
         createAndAppend('div', root, { text: err.message, class: 'alert-error' });
       } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
+        //createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
+        const select = createAndAppend('select', root);
+        createAndAppend('option', select, { text: 'Click here to choose a Repository' });
+        data.forEach(repo => {
+          const name = repo.name;
+          createAndAppend('option', select, { text: name });
+        });
+
+        const repoInfo = createAndAppend('div', forRepoBlock);
+        const contribs = createAndAppend('div', forContributorsBlock);
+        select.addEventListener('change', evt => {
+          const selectedRepo = evt.target.value;
+          const repo = data.filter(r => r.name == selectedRepo)[0];
+          console.log(repo);
+          repoInfo.innerHTML = '';
+          contribs.innerHTML = '';
+
+          const addInfo = (label, value) => {
+            const container = createAndAppend('div', repoInfo);
+            createAndAppend('span', container, { text: label });
+            createAndAppend('span', container, { text: value });
+          };
+          addInfo('Name: ', repo.name);
+          addInfo('Desciption: ', repo.description);
+          addInfo('Number of forks: ', repo.forks);
+          addInfo('Updated: ', repo.updated_at);
+
+          const contribsUrl = repo.contributors_url;
+          fetchJSON(contribsUrl, (err, contribData) => {
+            contribData.forEach(contributor => {
+              createAndAppend('p', contribs, { text: ':' });
```

Comment:
I find the output (html) of this a bit strange, I would expect that it was something like this:
```html
<div class="contributor">
  <span>contributor name</span>
  <img src="..." />
</div>
```